src/components/table/partial-styles/_row-selection.scss

```diff
@@ -0,0 +1,81 @@
+#tabulator-table {
+    &.has-rowselector {
+        .tabulator-frozen.tabulator-frozen-left {
```

Comment:
@anderssonjohan I'm not a big fan of targeting the cells with checkboxes, using this CSS. We're simply saying that if a cell is frozen and it's frozen to left, in a table with row-selectors, then apply these styles for that cell. This means if a consumer adds another frozen cell, these styles will affect them too.

Can we apply a specific and unique class to cells that contain the checkboxes? That'd be a better solution.

Reply:
for example, we may add a custom frozen cell to limel-table in future which enables keyboard navigation with arrow keys.

Reply:
For sure, this was used to match and override the styles provided by tabulator's default CSS.
Btw, all frozen columns will have `.tabulator-frozen` but only the one on the edge will have `tabulator-frozen-left`. See the frozen column module for reference on this: https://github.com/olifolkerd/tabulator/blob/c3c6bac2d739ff637897a7d51a79115f1c52d5e4/src/js/modules/FrozenColumns/FrozenColumns.js#L186-L188

I'm aware of the possibility of a future problem here, but I don't usually try to make my code aware of future challenges (not the same as writing future proof code!).

If you're saying we are about to use more frozen columns right now/in a near future I think we need to discuss more things, such as how and where these are injected to get them in the right order. 
Right now the table selection feature prepends its checkbox column to the existing set of columns (https://github.com/Lundalogik/lime-elements/pull/1514/files#diff-bab4217d5a11d0c96931f9e9f63bb15ed9564ee64b74eebd6927d68643ea6b49R42)

Reply:
Also, I don't think we are able to add custom css classes to the columns. My opinion is that the left-most (for default left to right text direction, which we only support for now) frozen column should be the row selector checkbox, thus this selector should be fine for now. If we decide to add more frozen columns we may need to skip using the tabulator-frozen-left class, which will select the right-most frozen-to-the-left column. Then we could select the first column with this frozen class instead. 

I'm happy to change this if we have more plans for frozen columns but for now I think we're good.

Reply:
Well, I hope we will do the example that I explained. But I don't know when. 

However, if we don't use frozen cells, someone internal or external who is using `limel-table` will do. (now I don't know who, but they may). Our baseline for Elements is that it is a public library and should not eff-up for others. If we were doing this in crm-components, I'd just write this as a comment in the CSS. But since it's elements, I think we should try to mitigate the potential problem now and make it as future-proof as we humanly can.

So if you can inject a class on those cells that contain the checkbox, we don't even have to use Tabulators STUPID frozen things that lag like heck due to their `absolute` positions, and dynamic calculations for `left`, as user scrolls left and right. We will just use a `sticky` position and we will be in control of future too.

Reply:
> Also, I don't think we are able to add custom css classes to the columns.

This was a false statement. Now I found the `cssClass` property of the column definitions which also adds the given class to the row cells (not only the header) in that column. Feel free to change the class name I used! :)

(Nice work on the sticky fix. Now it looks really good when scrolling!)

About frozen columns, no one CAN use it because we don't allow them too. So it's not true that a consumer of limel-table may use frozen columns until we also add support for it in limel-table's column specification. We only allow a subset of all the options that Tabulator provides.

Reply:
> no one CAN use it because we don't allow them

Gotcha! But we may allow that too. So better not make it hard for our future-selves :) 

Great job with adding the class! Now I can sleep well 😇